devDocs/developerGuide.t2t

```diff
@@ -12,6 +12,22 @@ NVDA NVDA_VERSION Developer Guide
 + Introduction +
 This guide provides information concerning NVDA development, including translation and the development of components for NVDA.
 
+++ API stability ++
+The NVDA API only includes symbols which are not marked with prefixing underscores.
+
+The NVDA Add-on API changes over time, such as removals, deprecations and new features.
+Important changes to the API are announced on the [NVDA API mailing list https://groups.google.com/a/nvaccess.org/g/nvda-api/about].
```

Comment:
Perhaps we should make it clear that changes to this policy will be conveyed via the changes file and mailing list.

```suggestion
Important changes to the API are announced on the [NVDA API mailing list https://groups.google.com/a/nvaccess.org/g/nvda-api/about].
Changes relevant to developers are also announced via the [NVDA changes file](https://www.nvaccess.org/files/nvda/documentation/changes.html).
Any changes to the API policy outlined in this section will be conveyed via these two channels.
```